sormas-api/src/main/resources/captions.properties

```diff
@@ -493,6 +493,8 @@ CaseHospitalization.previousHospitalizations=Previous hospitalizations
 CaseHospitalization.intensiveCareUnit=Stay in the intensive care unit
 CaseHospitalization.intensiveCareUnitStart=Start of the stay
 CaseHospitalization.intensiveCareUnitEnd=End of the stay
+CaseHospitalization.hospitalizationReason=Reason for hospitalization
+CaseHospitalization.otherHospitalizationReason=Specify other reason for hospitalization
```

Comment:
Change to "Specify reason", otherwise the caption will get too long (especially in languages like German)